pkg/klusterlet/clusterclaim/clusterclaimer.go

```diff
@@ -338,10 +338,14 @@ func (c *ClusterClaimer) getOCPVersion() (version, clusterID string, err error)
 
 	clusterID = string(clusterVersion.Spec.ClusterID)
 	historyItems := clusterVersion.Status.History
+	var latestCompleteTime *metav1.Time
+
 	for _, historyItem := range historyItems {
 		if historyItem.State == "Completed" {
-			version = historyItem.Version
-			break
+			if latestCompleteTime.IsZero() || latestCompleteTime.Before(historyItem.CompletionTime) {
+				latestCompleteTime = historyItem.CompletionTime
+				version = historyItem.Version
```

Comment:
Could we rename the v`ersion` to `latestCompleteVersion` to match the `latestCompleteTime`

Reply:
done